_data/browsers.yml

```diff
@@ -1,17 +1,23 @@
 - firefox:
   vendor: Mozilla
   name: Firefox
+  version: 77
 
 - chrome:
   vendor: Google
   name: Chrome
+  version: 85
 
 - edge:
   vendor: Microsoft
   name: Edge
+  version: 85
 
 - safari:
   vendor: Apple
   os: iOS
   name: Safari
-  version: 10.3
\ No newline at end of file
+  version: 13.4
```

Comment:
These are minimum versions. As replaceAll is not in the caniuse compiled data.json database at the moment, I had to hardcode the minimum versions here. Ah well.